include/seqan3/io/sequence_file/format_genbank.hpp

```diff
@@ -52,8 +52,8 @@ namespace seqan3
  *
  * ### Introduction
  *
- * genbank is the format used in GenBank sequence database. See
- * (http://quma.cdb.riken.jp/help/gbHelp.html) for a an in-depth description of the format.
+ * genbank is the format used in the GenBank sequence database. See [this example record at NCBI]
+ * (https://www.ncbi.nlm.nih.gov/Sitemap/samplerecord.html) for an in-depth description of the format.
```

Comment:
```suggestion
 * (https://www.ncbi.nlm.nih.gov/Sitemap/samplerecord.html) for more details about the format.
```
it is not an in-depth description if it is just a sample record?